File: sentimental-mario-less/mario.py
from cs50 import get_int

# Get the Height of the triangle
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    for j in range(height - i - 1):  # Push the pyramid to the right
        print(end=" ")
    for j in range(i + 1):  # Print the pyramid column by column
        print("#", end="")
    print()  # Start a new column

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    //pyramids height selector
    do
    {
        n = get_int("height: ");
    }
    while (n < 1 || n > 8);

    //pyramids maker
    for (int i = 0; i < n; i++)
    {
        //pusher that make the first pyramid right-facing
        for (int j = n; j > i + 1; j--)
        {
            printf(" ");
        }
        //pyramid right-facing "wall"
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //pyramids gap
        printf("  ");

        //pyramid left-facing "wall"
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //next line
        printf("\n");
    }
}